### DISCUSSION TITLE
Database "last updated" time

### DISCUSSION THREAD

**Discussion by zaneshaw** [pocketbase/pocketbase#3018] [Q&A] [answered]
Is there a streamlined way to fetch the last time a PocketBase database was last written using the Web API? Couldn't find anything just by Googling or looking through the docs.

**Comment by ganigeorgiev** [comment, marked as answer]
You can try ordering DESC by the `created` field. Something like:

```js
await pb.collection("example").getList(1, 1, {
  sort: "-created",
  skipTotal: true, // available with the next release
})
```

**Reply by zaneshaw** [reply]
Thanks! I would just need to run this with all the collections that I need and just find the latest date.

**Reply by ganigeorgiev** [reply]
I actually misunderstood what you want.

It seems that you want the last time the database was modified for **all collections** but we don't have web API for this.

I'm not familiar with your use case, but as you've mentioned you can manually fetch the last item for each collection and compare them (in your case maybe sorting by the `updated` field will be better), but keep in mind that this may not be very efficient if you have large number of collections.

**Reply by zaneshaw** [reply]
Yeah I've already been using this method during development and was just wondering if there was a built-in way to do this. Not too worried about efficiency as the database is only built for around ~20 users. Appreciate the quick response time though. :)
